### PR TITLE
MOVE: describe to pandas/core/describe.py

### PR DESCRIPTION
**Pull request by ivanovmg**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

Move ``describe`` functionality to a separate module and prepare for the further refactor/simplification (xref #36833).

 - Created new module ``pandas/core/describe.py``
 - Delegate the functionality of ``NDFrame.describe`` to function ``describe_ndframe`` in the new module

### CODE REVIEW

File: pandas/io/formats/describe.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
+"""
+Module responsible for execution of NDFrame.describe() method.
```
Comment (jreback) [inline pandas/io/formats/describe.py:2]: i would put in pandas/core/describe.py

**Review by jreback — CHANGES_REQUESTED** [review]
location suggestion. ping on green.

thanks for just *moving*.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
@ivanovmg actually can you merge master again, think i just merged an incompatible change here

**Comment by jreback** [comment]
great ping on green

**Comment by ivanovmg** [comment]
> great ping on green

@jreback, it is green.

**Comment by jreback** [comment]
thanks @ivanovmg

**Pull request merged**